todo.txt

```diff
@@ -0,0 +1,5 @@
+-save and return all-time best path
+-tune parameters
+-try different swapping strategies
```

Comment:
Descending linear swap added in PR 5 https://github.com/dekatzenel/cs205-final-project/pull/5